.github/workflows/deploy_prod.yml

```diff
@@ -46,8 +47,21 @@ jobs:
           echo "::set-output name=releaseChannel::$RELEASE_CHANNEL"
       - name: Install Packages
         run: npm install
+      - name: Get Latest Release # Release tag will be prod-<version> i.e. prod-3.7.1, prod-3.8.0
+        id: latestRelease
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: ${{ github.repository }}
```

Comment:
Hmm can we standardise with the branch preview one? Realised that in future when devs see this, they will likely be thinking why the difference.